### DISCUSSION TITLE
Error: "Params 'module' and 'action' are required parameters" on mud worlds API

### DISCUSSION THREAD

**Discussion by dev-jb** [blockscout/blockscout#11203] [Q&A] [answered]
I am trying to integrate mud on the explorer. I have configured and enable indexer on envs/common-blockscout.env as well:
**MUD_INDEXER_ENABLED**=true
**MUD_DATABASE_URL**=postgres://username:password@IP:5432/dbname
**MUD_POOL_SIZE**=50

Still getting the error from explorer while trying to query /api/v2/mud/worlds
<img width="636" alt="image" src="https://github.com/user-attachments/assets/b288e67a-cef8-497c-bbb4-382fc5436178">

I have explorer to the latest version as well:
<img width="315" alt="image" src="https://github.com/user-attachments/assets/acd03790-6b42-4ec0-998f-a9167e0a8074">

**Comment by nikitosing** [comment, marked as answer]
I suppose you should use `blockscout/blockscout-redstone` docker image, or rebuild locally with set `MUD_INDEXER_ENABLED=true`

**Reply by Duonget123** [reply]
{
  transaction(hash: "undefined") {
    hash
    blockNumber
    value
    gasUsed
  }
}

**Reply by dev-jb** [reply]
> I suppose you should use `blockscout/blockscout-redstone` docker image, or rebuild locally with set `MUD_INDEXER_ENABLED=true`

This worked! Thanks!
